tests/main.test.js

```diff
@@ -0,0 +1,10 @@
+// import { randNumber } from '@ngneat/falso';
+// import { sum } from 'src/scripts/main.js;
+//
+//
+// test('When adding new valid product, get successful confirmation', () => {
+//   const x = randNumber();
+//
+//   expect(sum(x, b)).toBe();
+//   expect(sum(x, b)).toBe();
+// });
```

Comment:
What about tests?
It is enough two tests: one for sum and one for getColor.

Reply:
I made it.